_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and natural language processing (NLP) support tools for OpenSearch. It provides data analysts with the following functionality:
+
+- Call OpenSearch indexes and manipulate using the opensearch-py-ml data frame. The opensearch-py-ml Data Frame wraps an OpenSearch index into a API similar to [pandas](https://pandas.pydata.org/), giving you the ability to process large amounts of data from OpenSearch inside a Jupyter Notebook.
+- Provides basic support to upload NLP [SentenceTransformers](https://www.sbert.net/) models into OpenSearch using the [ML Commons plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/).
+- Gives the ability to train and tune SentenceTransformer models with ML Common and synthetic queries.
```

Comment:
@mingshl Mingshi, is this part accurate? Do we need ML Commons when train and tune models?

Reply:
nope, I also commented above and we don't need  ML Commons when train and tune models

Reply:
I switched to, "...with synthetic queries".